example/src/App.svelte

```diff
@@ -8,6 +8,16 @@
     <li><a href="/does/not/exist" use:link>Not found</a></li>
 </ul>
 
+<h2>Dynamic links</h2>
```

Comment:
For a purely cosmetic reason, can you move this at the bottom of the page, below the router?

Reply:
Done :)